org._3fz.mtoc.metainfo.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<component type="desktop-application">
+  <id>org._3fz.mtoc</id>
+  <launchable type="desktop-id">org._3fz.mtoc.desktop</launchable>
+  <name>Mtoc</name>
+  <summary>Visual Music Library</summary>
+
+  <developer id="org._3fz">
+    <name>Asa DeGroff</name>
+  </developer>
+  <update_contact>asa@3fz.org</update_contact>
+
+  <description>
+    <p>Mtoc is a visually-rich music player and library browser that emphasizes a smooth, continuous experience with album artwork at the forefront.</p>
+    <ul>
+      <li>Animated, visual browsing made for music fans who enjoy flipping through albums</li>
+      <li>Library browsing with smart organization and responsive search</li>
+      <li>"Bento Box" layout places large-scale album art with playback controls and your library side by side</li>
+      <li>Advanced queue management with library context menus and drag-and-drop reordering</li>
+      <li>Save playlists from your queue, manage M3U files, and use the intuitive playlist editor</li>
+      <li>Directory picker and one-click scanning extracts track metadata</li> 
+      <li>Lightweight database layer stores tags and thumbnails to keep your library responsive</li>
+      <li>Desktop integration with full MPRIS 2 support</li>
+      <li>GStreamer-based playback engine with support for a wide variety of formats and gapless playback</li>
+    </ul>
+  </description>
+
+  <screenshots>
+    <screenshot type="default">
+      <caption>Main window showing library and now playing</caption>
+      <image>https://github.com/asa-degroff/mtoc/raw/main/resources/banner/main-screenshot.png</image>
```

Comment:
this (a below) should never reference the tip of a branch because it's not immutable.